test/scenarios/el94-src@rpm-install.sh

```diff
@@ -40,16 +37,12 @@ scenario_run_tests() {
     local -r source_reponame=$(basename "${LOCAL_REPO}")
     local -r source_repo_url="${WEB_SERVER_URL}/rpm-repos/${source_reponame}"
     local -r target_version=$(current_version)
-    local -r current_minor_version=$(current_minor_version)
-    local -r previous_minor_version=$(previous_minor_version)
-    local -r previous_version_repo=$(get_rel_version_repo "${previous_minor_version}")
-    local -r previous_version_repo_url=$(echo "${previous_version_repo}" | cut -d, -f2)
 
     # Enable the repositories with the dependencies.
 
     # RHOCP for current version might not exist yet. Then, just use whatever we can get for previous minor.
-    local -r rhocp_current=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${current_minor_version}" || true)
-    local -r rhocp_previous=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${previous_minor_version}")
+    local -r rhocp_current=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${MINOR_VERSION}" || true)
+    local -r rhocp_previous=$("${ROOTDIR}/scripts/get-latest-rhocp-repo.sh" "${PREVIOUS_MINOR_VERSION}")
```

Comment:
Would this work?

```suggestion
    local -r rhocp_current="${CURRENT_RELEASE_REPO}"
    local -r rhocp_previous="${PREVIOUS_RELEASE_REPO}"
```

and below:

```
-        if [[ "${rhocp}" =~ ^[0-9]{2} ]]; then
+        if [[ "${rhocp}" =~ ^rhocp ]]; then
```